src/Classes/Cli/TextRenderer.php

```diff
@@ -24,50 +24,39 @@ public static function color(string $text, int $color): string
         return "\e[" . $color . "m" . $text . "\e[0m";
     }
 
-    public static function rightPad(string $text, int $totalLength): string
+    public static function rightPad(string $text, int $length): string
     {
-        $textLength = strlen($text);
-
-        if ($textLength >= $totalLength) {
-            return $text;
-        }
-
-        $paddingLength = $totalLength - $textLength;
-        $padding = str_repeat(' ', $paddingLength);
-
-        return $text . $padding;
+        return \str_pad($text, $length, ' ', \STR_PAD_RIGHT);
     }
 
-    public static function renderHelpHeading(string $heading): string
+    public static function renderLogo()
     {
-        return self::color($heading, self::COLOR_YELLOW) . "\n";
+        // echo "    __  _____  _____    ______   ________    ____\n";
+        // echo "   /  |/  /  |/  / /   / ____/  / ____/ /   /  _/\n";
+        // echo "  / /|_/ / /|_/ / /   / /      / /   / /    / /  \n";
+        // echo " / /  / / /  / / /___/ /___   / /___/ /____/ /   \n";
+        // echo "/_/  /_/_/  /_/_____/\____/   \____/_____/___/   \n";
+        // created with: https://patorjk.com/software/taag/#p=display&f=Slant&t=MMLC%20CLI
+
+        echo "    __  ___ __  ___ __    ______   ______ __     ____\n";
+        echo "   /  |/  //  |/  // /   / ____/  / ____// /    /  _/\n";
+        echo "  / /|_/ // /|_/ // /   / /      / /    / /     / /  \n";
+        echo " / /  / // /  / // /___/ /___   / /___ / /___ _/ /   \n";
+        echo "/_/  /_//_/  /_//_____/\____/   \____//_____//___/   \n";
+        // cretated with: https://patorjk.com/software/taag/#p=display&h=1&f=Slant&t=MMLC%20CLI
```

Comment:
The TextRenderer no longer has this method renderLogo(). It is now included in MmlcCli.